client/nats/client.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package nats
```

Comment:
I don't know what I feel about this client package. It doesn't really provide any abstractions over `github.com/nats-io/nats.go`.

Reply:
The main reason is to allow mocking in unit tests. Do you have a better suggestion?

Reply:
Sorry, I didn't realize you were depending on the mocks as I merely glossed over the unit tests. Disregard this comment.